lib/rules/web/admin_console/4.19/common_ui_elements.xyaml

```diff
@@ -944,7 +944,7 @@ wait_table_loaded:
 skip_dev_perspective_tour:
   elements:
   - selector:
-      xpath: //button[@id='tour-step-footer-secondary' and text()='Skip tour']
```

Comment:
`//button[@data-test="tour-step-footer-secondary"]` or `//button[@id="tour-step-footer-secondary"]` is enough